aopy/visualization/base.py

```diff
@@ -418,12 +420,22 @@ def annotate_spatial_map_channels(acq_ch=None, drive_type='ECoG244', theta=0, co
             plt.axis('off')
 
         .. image:: _/images/ecog244_opto32.png
+
+    Note: 
+        The acq_ch returned from `func::aopy.data.load_chmap` are 1-indexed lists of acquisition 
+        channels connected to electrodes. However, the acq_ch input to this function is always 
+        0-indexed (or None for all channels). The print_zero_index argument controls in what format the
+        channels are printed onto the figure.
     '''
     if ax is None:
         ax = plt.gca()
     if acq_ch is not None:
         acq_ch = np.array(acq_ch)+1 # Change from 0 to 1 index
```

Comment:
change this to: 
```
if acq_ch is not None: 
       acq_ch = np.array(acq_idx) + 1
```

Reply:
plan is to have two different input variables for acq_ch vs acq_idx. That would be less confusing and will prevent unintentional -1 or +1.